docs/dashboards/markdown.md

```diff
@@ -0,0 +1,8 @@
+title: Markdown
+description: Free-form text with Markdown
+
+The Markdown component enables you to show free-form text using Markdown syntax. It is a tool for including helpful tips, describing how to analyze the data on dashboard, and including external links. 
+The links can point to your internal documentation or external websites.
+
+![Markdown Component](../images/dashboards/markdown-component.png)
+
```

Comment:
We could point out that adding an Event through a UI is example of a MD component.
https://sematext.com/docs/events/adding/#adding-events-through-ui

Reply:
updated: https://github.com/sematext/docs/blob/ful-dashboards/docs/images/events/custom-event.gif